idb/postgres/internal/writer/writer.go

```diff
@@ -379,6 +379,8 @@ func (w *Writer) addTransactionParticipation(block *bookkeeping.Block) error {
 	return nil
 }
 
+// Describes a change to the `account.keytype` column. If `present` is true,
+// `value` is the new value. Otherwise, NULL will be the new value.
```

Comment:
Thanks. This is still pretty confusing, but the comment helps. We probably should have added a `UKNOWN` SigType, but this works.